src/devtoolsPanel.ts

```diff
@@ -311,35 +312,42 @@ export class DevToolsPanel {
         // Convert the local url to a workspace path
         const transformer = new debugCore.UrlPathTransformer();
         void transformer.launch({ pathMapping: this.config.pathMapping });
-        const localSource = { path: sourcePath };
-        await transformer.fixSource(localSource);
 
-        sourcePath = localSource.path || sourcePath;
+        // origin in this case is trivial since we expect fixSource to take it out
+        const localSource = { path: sourcePath, origin: 'vscode-webview://' };
```

Comment:
nit: Since we expect this to be cleared, I suggest using a totally invalid/nonsense origin just in case, something like 'invalid-origin://' unless that breaks the transformer